src/mir/well-formed-mir.rkt

```diff
@@ -71,6 +86,27 @@
    (well-formed/Place Γ LocalId)
    ]
 
+  [(well-formed/Place Γ Place)
+   ----------------------------------------
+   (well-formed/Place Γ (* Place))
+   ]
+
+  [(well-formed/Place Γ Place)
+   ----------------------------------------
+   (well-formed/Place Γ (field Place _))
```

Comment:
Should I check the existence of the `FieldId` here or is this better handled during type-checking?

Reply:
Hmm. I think that's a type-check concern.